src/workloads/execution/TimeSeriesLastpoint.yml

```diff
@@ -3,9 +3,11 @@ Owner: "@mongodb/query"
 Description: |
   This test exercises the behavior of lastpoint-type queries on time-series collections. The
   currently supported lastpoint aggregate pipelines that are tested here include:
-    1. a $sort on a meta field (ascending) and time (descending) and $group with _id on the meta
+    1. a $sort on a meta field (ascending, descending) and time (descending) and $group with _id on the meta
```

Comment:
nit: The meta phrasing is a little confusing, maybe something like `meta field (both ascending and descending)`?